src/state/reducerActions.ts

```diff
@@ -0,0 +1,32 @@
+import { Movie } from '../Types/MovieTypes';
+
+enum reducerActions {
+  HANDLE_MOVIES = 'HANDLE_MOVIES',
+  HANDLE_ERROR = 'HANDLE_ERROR',
+  HANDLE_LOADING = 'HANDLE_LOADING',
+  HANDLE_FILTER = 'HANDLE_FILTER',
+  HANDLE_SORT = 'HANDLE_SORT',
+  HANDLE_OFFSET = 'HANDLE_OFFSET',
+}
+
+type moviesJSON = {
```

Comment:
Type names should start with a capital letter. Moreover, removing JSON from the type's name is better because the return type may change, for example, to XML. So, I suggest MoviesDTO.

Reply:
Changed type to MoviesDTO.